.github/workflows/error-ref-publisher.yaml

```diff
@@ -0,0 +1,55 @@
+name: Error Code Reference Publisher
+on:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  error-ref:
+    name: Error codes utility
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # token here with write access to meshkit repo
+        with:
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+
+      - name: Setup Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ secrets.GO_VERSION }}
+
+      - name: Run utility
+        run: |
+          make errorutil-analyze
+      # to update errorutil* files in meshkit repo
+      - name: Commit changes
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_user_name: l5io
+          commit_user_email: ci@layer5.io
+          commit_author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+          commit_options: '--signoff'
+          commit_message: "run error codes utility"
+          file_pattern: helpers/
+
+      # to push changes to meshery docs
+      - name: Checkout meshery
+        uses: actions/checkout@v2
+        with:
+          repository: 'meshery/meshery'
+          # token with write access to meshery repository
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+          path: 'meshery'
+          ref: 'master'
+
+      - name: Update and push docs
+        run: |
+          echo '{ "errors_export": "" }' | jq --slurpfile export ./helpers/errorutil_errors_export.json '.errors_export = $export[0]' > ./meshery/docs/_data/errorref/meshkit_errors_export.json
```

Comment:
@DelusionalOptimist will this line become 
`echo '{ "errors_export": "" }' | jq --slurpfile export ./helpers/errorutil_errors_export.json '.errors_export = $export[0]' > ./meshery/docs/_data/errorref/meshery-osm_errors_export.json`

Reply:
I guess we'll need to do
```
echo '{ "errors_export": "" }' | jq --slurpfile export ./helpers/errorutil_errors_export.json '.errors_export = $export[0]' > ./meshery/docs/_data/errorref/osm_errors_export.json
```
that's how we're doing it in https://github.com/meshery/meshery/blob/master/docs/_data/errorref/kuma_errors_export.json

Reply:
got it